docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,370 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "AtomDB is tailored for accessing atomic properties and, in some cases, leveraging them to obtain approximate molecular properties (pro-molecule models).  The library is designed to be lightweight and easy to use with the goal of providing first approximation when the cost of more accurate calculations is prohibitive (e.g. high size of the system like in proteins with thousands of atoms or in cases where there are thousands of different atomic species like in high-throughput screening or molecular dynamics simulations). This notebook provides a quick overview of the main features of AtomDB and how to use it as a Python script or library.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use as a Python script\n",
+    "\n",
+    "AtomDB supports usage as a Python script for obtaining properties from the command line. While the script is not as feature-rich as the library, it is still useful for quick fetching of atomic properties. The next cell demonstrates how to utilize AtomDB as a Python script to retrieve atom properties.\n",
+    "\n",
+    "Note that to execute it (in a command line environment), you'll need to remove the ! from the beginning of the lines.\n",
+    "\n",
+    "**Getting help on the script**: In the following cell, we show how to get help on the script."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "usage: atomdb [-h] (-c | -q) [-e EXC] dataset elem charge mult\n",
+      "\n",
+      "Compile or query an AtomDB entry.\n",
+      "\n",
+      "options:\n",
+      "  -h, --help         show this help message and exit\n",
+      "\n",
+      "commands:\n",
+      "  -c, --compile      compile a species into the database\n",
+      "  -q, --query        query a species from the database\n",
+      "\n",
+      "arguments:\n",
+      "  dataset            name of dataset\n",
+      "  elem               element symbol\n",
+      "  charge             charge\n",
+      "  mult               multiplicity\n",
+      "  -e EXC, --exc EXC  excitation level\n"
+     ]
+    }
+   ],
+   "source": [
+    "!python -m atomdb -h"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Printing atom properties using the script:**\n",
+    "\n",
+    "The data is printed as human readable following the JSON format. This can be latter parsed and used \n",
+    "in other scripts or programs. The following cell shows how to print the data for the Carbon atom with\n",
+    "charge 0 and spin multiplicity 3 (ground state). Due to the large amount of data, the output stored\n",
+    "in the variable c_0_3_json_output sn only three properties are parsed and printed."
```

Comment:
This last sentence looks strange to me.
Could it be:
Due to the large amount of data, from the output stored in the variable c_0_3_json_output only three properties are parsed and printed.

Reply:
Done! Thanks!!